docs/time_units.md

```diff
@@ -281,16 +292,38 @@ SLiM also keeps track of "how many birth phases of the life cycle have happened
 (the column "# births" in the tables).
 As the simulation goes along,
 tskit time ago is recorded as minus one times the number of birth phases so far.
+(We multiply by -1 because as we move forward in time,
+we accumulate a negative number of "time-ago" units.)
 When the tree sequence is output, the current cumulative number of birth phases
 is added to this,
 so "tskit time ago" is, equivalently, "how many birth phases happened since this time".
 In a nonWF model, the two counters ("tick" and "number of birth phases")
-are always in sync; but in a WF they are not (during *first* and *early*).
+are always in sync; but in a WF model they are not (during *first* and *early*).
 The extra wrinkle this introduces is that the correspondence between "tskit time ago"
 and "SLiM time" depends on *which phase the tree sequence was recorded in*.
 
+When the tree sequence is written out, SLiM records the value of its current tick
+and the current stage,
+which can be found in the metadata: ``ts.metadata['SLiM']['tick']``,
+and ``ts.metadata['SLiM']['stage']``.
+The "SLiM time" referred to by a ``time`` in the tree sequence
+(i.e., the value that would be reported by ``community.tick``
+within SLiM at the point in time thus referenced)
+can be obtained by subtracting ``time`` from ``ts.metadata['SLiM']['tick']``
+and possibly adding or subtracting one depending on the current stage.
+The fiddly details are worked out by the {func}`.slim_time` method,
+which takes a ``stage`` argument.
+
+Some of the other methods in pyslim -- those that depend on {func}`.individuals_alive_at`
+-- need you to tell them during which stage the tree sequence was saved with ``sim.treeSeqOutput``,
+and need this to be the same as the stage that any individuals were saved with ``sim.treeSeqRememberIndividuals``.
+(This can add another factor of plus or minus one to the calculations.)
+This argument, ``remembered_stage``, defaults to *late()*;
+we recommend that you also default to always Remembering individuals, and saving out the tree sequence,
+during *late()* as well, unless you have good reason not to.
+
```

Comment:
what follows below is technical enough that you should probably add it as reference subsection (clickable on the RHS of the page) and maybe call it something like "Details of SLiM time conversion" or something similar